src/main/resources/templates/login.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <title>Nivio login page</title>
+    <style>
+
+        body {
+            margin: 0;
+            width: 100vw;
+            height: 100vh;
+            display: flex;
+            align-items: center;
+            text-align: center;
+            justify-content: center;
+            place-items: center;
+            overflow: hidden;
+            font-family: Monospace, serif;
+            background: #[[@{(${brandingBackground})}]];
+        }
+
+        .container {
+            display: flex;
+            flex-direction: column;
+            position: relative;
+            border-radius: 5px;
+            padding: 40px;
+            box-sizing: border-box;
+            box-shadow: 4px 4px 2px #0d0d0e;
+            background: #[[@{(${brandingForeground})}]];
+        }
+
+        .welcome {
+            color: #22F2C2;
+        }
+
+        .brand-logo {
+            height: 65px;
+            width: 65px;
+            background: #A0A0A7;
+            margin: auto;
+            border-radius: 50%;
+            box-sizing: border-box;
+        }
+
+        .brand-title {
+            margin-top: 10px;
+            font-weight: 900;
+            font-size: 1.8rem;
+            color: #A0A0A7;
+            letter-spacing: 1px;
+        }
+
+    </style>
+</head>
+
+<body>
+
+<div class="container">
+    <div class="nivio-logo"><img src="/icons/svg/nivio.svg" alt="nivio logo" width="80" height="80"></div>
+    <div class="welcome"><h2 th:text="${brandingMessage}"></h2><h3>please login</h3><br></div>
+    <div class="brand-logo">
+        <a th:if="${auth_github} != null" th:href="@{(${auth_github.href})}">
+            <img src="/icons/svg/github.svg" alt="github logo" width="65" height="65">
+        </a>
+    </div>
+    <div class="brand-title">GITHUB</div>
```

Comment:
I forgot to move this inside the th:if blockl

Reply:
I do not understand 100 %, maybe we can talk tomorrow about it.